Gems/AWSGameLift/Code/AWSGameLiftServer/Source/AWSGameLiftServerManager.cpp

```diff
@@ -452,9 +486,43 @@ namespace AWSGameLift
             return;
         }
 
-        AZ_TracePrintf(AWSGameLiftServerManagerName, "Initiating Amazon GameLift Server SDK ...");
-        Aws::GameLift::Server::InitSDKOutcome initOutcome = m_gameLiftServerSDKWrapper->InitSDK();
-        AZ_TracePrintf(AWSGameLiftServerManagerName, "InitSDK request against Amazon GameLift service is complete.");
+        Aws::GameLift::Server::Model::ServerParameters serverParameters;
+
+        if (sv_gameliftAnywhereEnabled)
+        {
+            if (auto webSocketUrl = static_cast<AZ::CVarFixedString>(sv_gameliftAnywhereWebSocketUrl); !webSocketUrl.empty())
+            {
+                serverParameters.SetWebSocketUrl(webSocketUrl.c_str());
+            }
+            if (auto authToken = static_cast<AZ::CVarFixedString>(sv_gameliftAnywhereAuthToken); !authToken.empty())
+            {
+                serverParameters.SetAuthToken(authToken.c_str());
+            }
+            if (auto fleetId = static_cast<AZ::CVarFixedString>(sv_gameliftAnywhereFleetId); !fleetId.empty())
+            {
+                serverParameters.SetFleetId(fleetId.c_str());
+            }
+            if (auto hostId = static_cast<AZ::CVarFixedString>(sv_gameliftAnywhereHostId); !hostId.empty())
+            {
+                serverParameters.SetHostId(hostId.c_str());
+            }
+            if (auto processId = static_cast<AZ::CVarFixedString>(sv_gameliftAnywhereProcessId); !processId.empty())
+            {
+                serverParameters.SetProcessId(processId.c_str());
+            }
+            else
+            {
+                // If ProcessId isn't defined, provide a unique string by default.
+                AZStd::string defaultProcessId = AZStd::string::format("ProcessId_%i", std::time(nullptr));
+                serverParameters.SetProcessId(defaultProcessId.c_str());
+                AZ_TracePrintf(
+                    AWSGameLiftServerManagerName, "Generated default Amazon GameLift ProcessId value: %s\n", defaultProcessId.c_str());
+            }
+        }
+
+        AZ_TracePrintf(AWSGameLiftServerManagerName, "Initiating Amazon GameLift Server SDK ...\n");
+        Aws::GameLift::Server::InitSDKOutcome initOutcome = m_gameLiftServerSDKWrapper->InitSDK(serverParameters);
+        AZ_TracePrintf(AWSGameLiftServerManagerName, "InitSDK request against Amazon GameLift service is complete.\n");

```

Comment:
Should break out the messaging depending on the outcome
```suggestion
        if (initOutcome.IsSuccess())
        {
            AZ_TracePrintf(AWSGameLiftServerManagerName, "InitSDK request against Amazon GameLift service is complete.\n");
        }
        else
        {
             AZ_TracePrintf(AWSGameLiftServerManagerName, "InitSDK request against Amazon GameLift service failed.\n");
        }

```

Reply:
Also AZ_TracePrintf has been [deprecated](https://github.com/o3de/o3de/issues/14838) for AZ_Trace
```suggestion
  if (initOutcome.IsSuccess())
        {
            AZ_Trace(AWSGameLiftServerManagerName, "InitSDK request against Amazon GameLift service is complete.\n");
        }
        else
        {
             AZ_Trace(PAWSGameLiftServerManagerName, "InitSDK request against Amazon GameLift service failed.\n");
        }

```

Reply:
Fixed, thanks!